tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -28,7 +28,7 @@ fi
 if [ "$INSTALLED_PYTHON_VERSION" = "3.4" ];then
   echo "Python 3.5 and above is needed for running onnx tests!" 1>&2
 else
-  source /tmp/scripts/install_onnx.sh $INSTALLED_PYTHON_VERSION
+  source ./install_onnx.sh $INSTALLED_PYTHON_VERSION
```

Comment:
If you run this script from a different folder, this will fail.

Reply:
Let's say you have `a.sh` and `b.sh` in folder `/home/myfolder/foo`, and `a.sh` calls `./b.sh`. If you run `a.sh` in folder `/home/another/folder`, the call will fail with file not found.

Reply:
let's use caller path then.